.github/workflows/typos.yaml

```diff
@@ -10,5 +10,5 @@ jobs:
       uses: actions/checkout@v4
 
     - name: Check spelling of file.txt
-      uses: crate-ci/typos@master
+      uses: crate-ci/typos@v1.19.0
```

Comment:
nit: we could of course find this info again via eg. `git blame`, but leaving a comment here explaining the choice for this specific version would make sense. WDYT?

Reply:
We do instruct dependabot to update our github actions: https://github.com/rancher/fleet/blob/main/.github/dependabot.yaml#L38-L44

So, eventually we would get a green PR?

Reply:
Anyway... I've added a note.
My plan is to re-test this next week as `1.20.1` was released fast after `1.20.0` and I suspect more releases will come shortly. 
It looks like `1.20.0` introduced way to many false positives.